configurations/leap_s06_pipeline.py

```diff
@@ -0,0 +1,252 @@
+                    ),
+                ]
+            ),
+            AnalysisDatasetConfiguration(
+                engagement_db_datasets=["kakuma_preffered_language"],
+                dataset_type=DatasetTypes.DEMOGRAPHIC,
+                raw_dataset="preffered_language_raw",
+                coding_configs=[
+                    CodingConfiguration(
+                        code_scheme=load_code_scheme("demographics/preffered_language"),
+                        analysis_dataset="preffered_language"
+                    ),
+                ]
+            ),
+            AnalysisDatasetConfiguration(
+                engagement_db_datasets=["kakuma_nationality"],
+                dataset_type=DatasetTypes.DEMOGRAPHIC,
+                raw_dataset="nationality_raw",
+                coding_configs=[
+                    CodingConfiguration(
+                        code_scheme=load_code_scheme("demographics/nationality"),
+                        analysis_dataset="nationality"
+                    ),
+                ]
+            ),
+            AnalysisDatasetConfiguration(
+                engagement_db_datasets=["kakuma_disabled"],
+                dataset_type=DatasetTypes.DEMOGRAPHIC,
+                raw_dataset="disabled_raw",
+                coding_configs=[
+                    CodingConfiguration(
+                        code_scheme=load_code_scheme("demographics/disabled"),
+                        analysis_dataset="disabled"
+                    ),
+                ]
+            ),
+            AnalysisDatasetConfiguration(
+                engagement_db_datasets=["kakuma_age"],
+                dataset_type=DatasetTypes.DEMOGRAPHIC,
+                raw_dataset="age_raw",
+                coding_configs=[
+                    CodingConfiguration(
+                        code_scheme=load_code_scheme("demographics/age"),
+                        analysis_dataset="age"
+                    ),
+                    CodingConfiguration(
+                        code_scheme=load_code_scheme("demographics/age_category"),
+                        analysis_dataset="age_category",
+                        age_category_config=AgeCategoryConfiguration(
+                            age_analysis_dataset="age",
+                            categories={
+                                (10, 14): "10 to 14",
+                                (15, 17): "15 to 17",
+                                (18, 35): "18 to 35",
+                                (36, 54): "36 to 54",
+                                (55, 99): "55 to 99"
+                            }
+                        )
+                    ),
+                ],
+            ),
+        ],
+        ws_correct_dataset_code_scheme=load_code_scheme("ws_correct_dataset"),
+    ),
+    archive_configuration=ArchiveConfiguration(
+        archive_upload_bucket="gs://pipeline-execution-backup-archive",
+        bucket_dir_path="2022/POOL-WUSC/"
```

Comment:
I think if you have a trailing '/' the files will upload to `...POOL-WUSC//data-...`. The double '/' makes listing the contents a bit trickier (as an aside, we should probably update the relevant pipeline stage to trim trailing '/'s so we don't have to think about this in future)

Reply:
noted